azext_iot/iothub/commands_certificate.py

```diff
@@ -0,0 +1,50 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from typing import Optional
+from azext_iot.iothub.providers.certificate import CertificateProvider
+from knack.log import get_logger
+
+logger = get_logger(__name__)
+
+
+def certificate_root_authority_show(
+    cmd,
+    hub_name: Optional[str] = None,
+    resource_group_name: Optional[str] = None,
+    login: Optional[str] = None,
```

Comment:
if this is control plane only functionality, `login` and `auth_type_dataplane` should be removed as function params.